### PR TITLE
feat: add four new stat variables, stake simulator

### PR DESCRIPTION
**Pull request by kemuru**
## PR-Codex overview
This PR focuses on enhancing the `CourtDetails` component and related functionalities by introducing new features, improving styling, and optimizing data handling for juror stakes and statistics.

### Detailed summary
- Added `effectiveStake`, `staked`, `locked`, and `delayed` fields in the GraphQL schema.
- Introduced a new `Divider` component for consistent styling.
- Updated various components to use the new `Divider`.
- Enhanced stake calculations for jurors.
- Improved data fetching and manipulation in hooks.
- Added new SVG icons for better visual representation.
- Implemented a `QuantityToSimulate` component for displaying staking simulations.
- Refactored `Stats` component to include new statistics related to PNK and ETH rewards.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

## Release Notes

- **New Features**
  - Introduced a new `SimulatorPopup` component to display statistical information related to juror staking.
  - Enhanced `Stats` component to display additional court performance statistics, improved number formatting, and added a time range selector.
  - Updated the `StatDisplay` component to support flexible title types and made the subtext property optional.
  - Added a new `Info` component to provide contextual information about past performance.
  - Introduced `QuantityToSimulate` and `Header` components for improved simulation interface.
  - Added a new `Divider` component for consistent visual separation across various components.

- **Bug Fixes**
  - Removed conditional SVG dimensions in the `StatDisplay` component to ensure consistent rendering.

- **Enhancements**
  - Expanded `jurorStakeDetailsQuery` to include `effectiveStake` for more comprehensive juror data.
  - Improved responsiveness and styling of the `InputDisplay` component for better user experience.
  - Streamlined the `StakePanel` component by removing unused elements and improving the user interface with the new `SimulatorPopup`.
  - Enhanced styling and layout adjustments across various components for improved visual appeal.

### CODE REVIEW

File: web/src/pages/Courts/CourtDetails/Stats.tsx [hunk 1]
```diff
@@ -116,26 +185,148 @@
...
+                  );
+                })}
+              </StyledCard>
+            </>
+          ),
+        },
+      ]}
+    ></StyledAccordion>
   );
 };
```
[the remark this answers is not in the document]
Comment (kemuru) [reply web/src/pages/Courts/CourtDetails/Stats.tsx]: i know

**Review by jaybuidl — APPROVED** [review]

File: web/src/pages/Courts/CourtDetails/index.tsx [hunk 2]
```diff
@@ -77,6 +77,14 @@ const StyledBreadcrumb = styled(Breadcrumb)`
   }
 `;
 
+const Divider = styled.hr`
```
Comment (Harman-singh-waraich) [inline web/src/pages/Courts/CourtDetails/index.tsx:80]: I think we use Divider in numerous places, can we extract it to components? 🤔

File: web/src/pages/Courts/CourtDetails/StakePanel/SimulatorPopup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+      ? courtCurrentEffectiveStake * currentTreeVotesPerPnk
+      : undefined;
+  const totalCases =
+    !isUndefined(courtCurrentEffectiveStake) && !isUndefined(currentTreeDisputesPerPnk)
+      ? courtCurrentEffectiveStake * currentTreeDisputesPerPnk
+      : undefined;
+  const totalRewards =
+    !isUndefined(courtCurrentEffectiveStake) && !isUndefined(currentTreeExpectedRewardPerPnk)
+      ? courtCurrentEffectiveStake * currentTreeExpectedRewardPerPnk
+      : undefined;
```
[the remark this answers is not in the document]
Comment (Harman-singh-waraich) [reply web/src/pages/Courts/CourtDetails/StakePanel/SimulatorPopup/index.tsx:178]: Agree 👍

File: web/src/pages/Courts/CourtDetails/Stats.tsx [hunk 4]
```diff
@@ -40,6 +97,33 @@ const StyledCard = styled.div`
...
+  margin-right: 16px;
+  small {
+    color: ${({ theme }) => theme.primaryText};
+  }
+  svg {
+    fill: ${({ theme }) => theme.primaryText};
+  }
+`;
+
+function beautifyStatNumber(value: number): string {
```
Comment (Harman-singh-waraich) [inline web/src/pages/Courts/CourtDetails/Stats.tsx:110]: There was a similar function in SimulatorPopup. We can extract this to utils.

**Review by Harman-singh-waraich — DISMISSED** [review]
Lgtm, Simulator looks awesome 🔥

**Review by jaybuidl — APPROVED** [review]

**Pull request merged**